bin/get-video-s3-acls.py

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env python3
+# Permissions required:
+#  psql - access to heroku app's postgres
+#  aws s3 - NZSL IAM access
+#  s3:GetObjectAcl permissions or READ_ACP access to the object
+#  https://docs.aws.amazon.com/cli/latest/reference/s3api/get-object-acl.html
+
+
+import os
+import sys
+import subprocess
+import argparse
+import json
+
+# TODO
+# We are using external apps just for the moment.
+# These will be removed for native libraries.
+AWSCLIENT = "/usr/local/bin/aws"
+PGCLIENT = "/usr/bin/psql"
+
+# NZSL: Is there a database url defined in the environment?
+DATABASE_URL = os.getenv("DATABASE_URL", None)
+if not DATABASE_URL:
+    print("You must define DATABASE_URL in the environment.", file=sys.stderr)
+    exit()
+
+# AWS: Is there an AWS_PROFILE defined in the environment?
+AWS_PROFILE = os.getenv("AWS_PROFILE", None)
+if not AWS_PROFILE:
+    print(
+        "You must define AWS_PROFILE in the environment. Eg. AWS_PROFILE='nzsl'",
+        file=sys.stderr,
+    )
+    exit()
```

Comment:
issue: this check defeats the whole point of the change - we want to leave it up to the user to have correctly configured their AWS credentials, which they can do using [any of the standard env variables the cli supports](https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-envvars.html)

Reply:
Post slack discussion: problem is this is over-specifying the auth method, so should be removed.

Reply:
Done.